### PR TITLE
[WIP] cassandra: Clean-up removing old examples

### PR DESCRIPTION
**Pull request by chrislovecnm**
I am working on getting some changes pushed to our container.  Once that is done I can test with a stable container.

Fixes https://github.com/kubernetes/examples/issues/10, https://github.com/kubernetes/examples/issues/7

TODO

- [ ] add new container to manifest
- [ ] update docs with new container
- [ ] some testing

### CODE REVIEW

**Comment by ahmetb** [comment]
@chrislovecnm I am not sure we're good with deleting the example altogether. I have an active plan to move the README.md of this example to the Kubernetes.io website (#12 and https://github.com/kubernetes/kubernetes.github.io/issues/4134) and applying this patch would remove this tutorial temporarily.

Instead of removing this example first, I would much rather prefer you just patch it.

**Comment by chrislovecnm** [comment]
So I am not removing the example, just portions of the example that are no longer functioning, and are not recommended.  The DS example is broken, and with StatefulSets I would never recommend running a DS.  Cassandra does not like when mulitple instances start at the same time.

I am happy to leave this open until you merge, but really we should update this example before it is on our website.

**Comment by ahmetb** [comment]
My bad,  [the rendering](https://user-images.githubusercontent.com/159209/27460564-79527c72-5769-11e7-9b14-668d74df453c.png) tricked me. I'll review.

File: cassandra/README.md [hunk 1]
```diff
@@ -420,433 +371,16 @@ $ grace=$(kubectl get po cassandra-0 --template '{{.spec.terminationGracePeriodS
...
-and `HEAP_NEWSIZE`), and adding information about the
-[namespace](../../../docs/user-guide/namespaces.md).
-We also tell Kubernetes that the container exposes
-both the `CQL` and `Thrift` API ports.  Finally, we tell the cluster
-manager that we need 0.1 cpu (0.1 core).
+[1]: https://kubernetes.io/docs/concepts/workloads/pods/pod/
+[2]: https://kubernetes.io/docs/concepts/services-networking/service/
+[3]: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+[Cassandra]: [http://cassandra.apache.org/]
+[container registry]: [https://quay.io/repository/vorstella/cassandra-k8s]
```
Comment (ahmetb) [inline cassandra/README.md]: I can help you update the `gcr.io/google-samples/cassandra` image if necessary, if you check in the Dockerfile.

Currently with this patch, we won't have the Dockerfile and you'll be the only one who has write access to the image.

**Comment by ahmetb** [comment]
@chrislovecnm I just sent #16 moving the tutorial (README.md) to the `kubernetes/kubernetes.github.io` repository repository. (https://github.com/kubernetes/kubernetes.github.io/pull/4179)

I saw that you have some non-trivial changes in README.md, we probably need to apply those to the `kubernetes/kubernetes.github.io` repository now.

**Comment by ahmetb** [comment]
@chrislovecnm sorry for the back and forth here. We decided to keep the `cassandra/README.md` in this repo, but I made some minor changes, so it renders on the website correctly at its new home: https://kubernetes.io/docs/tutorials/stateful-application/cassandra/ 

Do you mind resolving the merge conflicts when you have time?

**Comment by chrislovecnm** [comment]
Yah no worries. I think I have a base container that is happy now.

Are we leaving the container in gcr.io or can I use one that I maintain?

**Comment by ahmetb** [comment]
Let's leave it on gcr.io. I can easily update it if you have a new Dockerfile.

**Comment by ahmetb** [comment]
Let's update references to `gcr.io/google-samples/cassandra:v13` and I'll make it available if you can check in the Dockerfile back.

**Comment by ahmetb** [comment]
#38 made some improvements to Cassandra tutorial, it would be great if we can apply the changes  suggested above to that.

**Comment by i7clock** [comment]
I figured that the current example is meant to work with 3.10, which is not available anymore... bump it to 3.11.0 in the make file? Otherwise building the image fails!

**Pull request closed without merging**